mambaSharedFramework/HLS Models/HLSPlaylistCore.swift

```diff
@@ -162,7 +162,11 @@ public struct HLSPlaylistCore<T>: HLSPlaylistInterface, CustomDebugStringConvert
     }
     
     public var playlistCoreDebugDescription: String {
-        return "registeredTags:\(registeredTags) \nstructure:\(structure)\n"
+        guard let stream = try? self.write(),
+            let debugDescription = String(data: stream, encoding: .utf8) else {
+            return "registeredTags:\(registeredTags) \nstructure:\(structure)\n"
```

Comment:
I feel like it would be helpful to have some indication that the has write failed here? That's probably going to be useful to the user in case they run into this.

```
return "Stream write failure. Raw Data: registeredTags:\(registeredTags) \nstructure:\(structure)\n"
```

Reply:
Good idea. Added.